### PR TITLE
Fixes #1341

### LINKED ISSUE #1341: Shipping country in the cart page is not sorted alphabetically
**Description**  
The title says it all. 

**Possible Solution**  
A simple sortBy and sortDirection maybe?

### PR DESCRIPTION
**Pull request by zaifastafa**
### 5. Checklist

- [ ] I have written tests and verified that they fail without my change
- [x] I have squashed any insignificant commits
- [x] I have created a [changelog file](https://github.com/shopware/platform/blob/master/adr/2020-08-03-Implement-New-Changelog.md) with all necessary information about my changes
- [ ] I have written or adjusted the documentation according to my changes
- [ ] This change has comments for package types, values, functions, and non-obvious lines of code
- [x] I have read the contribution requirements and fulfil them.

### CODE REVIEW

**Comment by pweyck** [comment]
A changelog file would be awesome :)

**Comment by zaifastafa** [comment]
> A changelog file would be awesome :)

Done! :+1:

File: changelog/_unreleased/2020-09-21-fix-country-sorting-in-cart.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+issue: NA
+author: Huzaifa Mustafa
+author_email: 24492269+zaifastafa@users.noreply.github.com 
+author_github: zaifastafa
+---
+# Core
+*  Added criteria in `\Shopware\Storefront\Page\Checkout\Cart\CheckoutCartPageLoader::getCountries` to first sort by 
+position and then by name, so the countries are sorted properly.   
+___
+# API
```
Comment (mitelg) [inline changelog/_unreleased/2020-09-21-fix-country-sorting-in-cart.md]: remove the the things you don't need

**Review by mitelg — COMMENTED** [review]
please squash your commits ☺️

**Comment by zaifastafa** [comment]
> please squash your commits

I think there is some issue with the squashing as the history is not getting rewritten and is giving conflicts. Should I delete the branch/PR and create a new one?

**Pull request closed without merging**